src/components/atoms/_OcGhostElement/_OcGhostElement.vue

```diff
@@ -20,6 +29,10 @@ export default {
     },
   },
   computed: {
+    iconStackCount() {
+      const count = this.previewItems.length
+      return count > 3 ? 3 : count
```

Comment:
Math.max maybe ;)

Reply:
yes :D right

Reply:
We might also at a variable with a descriptive name for the magic number 3 ;)

Reply:
yes will do so